Mage/src/main/java/mage/filter/common/FilterSuspendedCard.java

```diff
@@ -0,0 +1,72 @@
+package mage.filter.common;
+
+import mage.MageObject;
+import mage.abilities.Ability;
+import mage.abilities.keyword.SuspendAbility;
+import mage.cards.Card;
+import mage.counters.CounterType;
+import mage.filter.FilterCard;
+import mage.filter.FilterImpl;
+import mage.filter.FilterInPlay;
+import mage.filter.predicate.mageobject.AbilityPredicate;
+import mage.game.Game;
+
+import java.util.UUID;
+
+/**
+ * @author emerald000
+ */
+public class FilterSuspendedCard extends FilterImpl<MageObject> implements FilterInPlay<MageObject> {
```

Comment:
I assume you based this off of FilterPermanentOrSuspendedCard, which needs the two internal filters, but this doesn't need that extra layer. Instead, just extend FilterCard and add the necessary predicates.

> A suspended card is a card that (1) has suspend, (2) is in exile, and (3) has one or more time counters on it.